src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_Synthesized.cs

```diff
@@ -590,6 +590,53 @@ public override async Task<int> GetIntAsync(int x)
                 Assert.Empty(parameter.GetAttributes());
             });
         }
+
+        [Theory]
+        [MemberData(nameof(OptimizationLevelTheoryData))]
+        [WorkItem(38801, "https://github.com/dotnet/roslyn/issues/38801")]
+        public void BaseMethodWrapper_DoNotInheritAttributes_TypeParameter(OptimizationLevel optimizationLevel)
```

Comment:
>BaseMethodWrapper_DoNotInheritAttributes_TypeParameter [](start = 20, length = 54)

Is this test failing without the change in SubstitutedTypeParameterSymbol?

Reply:
Yes